vagrant/provisioning/roles/alfresco7/tasks/apply-amp.yml

```diff
@@ -1,3 +1,10 @@
+- name: Change ownership of alfresco.war
+  ansible.builtin.file:
+    path: /opt/app/arkcase/app/alfresco7/webapps/alfresco.war
+    owner: alfresco7
+    group: alfresco7
+  become: yes
```

Comment:
become is not here, is at the top of the task where we declare the executor of the task

Reply:
Okay Martin!